<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libro2</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="handleWidth">
         <number>3</number>
        </property>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BookInfoPanel" name="bookInfo" native="true">
            <property name="minimumSize">
             <size>
              <width>290</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>290</width>
              <height>0</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="BookTableView" name="bookList">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameFilter">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="textFilter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolFilterButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/right_14px.png</normaloff>:/icons/right_14px.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_file"/>
    <addaction name="actionAdd_folder"/>
    <addaction name="separator"/>
    <addaction name="actionSave_metadata"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="actionConvert"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelect_all"/>
    <addaction name="actionRemove_selected_files"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuToolbar_icon_size">
     <property name="title">
      <string>Toolbar icon size</string>
     </property>
     <addaction name="actionToolbarIconSmall"/>
     <addaction name="actionToolbarIconsLarge"/>
    </widget>
    <addaction name="actionViewInfo_panel"/>
    <addaction name="actionFilter_panel"/>
    <addaction name="actionViewToolbar"/>
    <addaction name="separator"/>
    <addaction name="menuToolbar_icon_size"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionForum"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="MainToolbar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_metadata"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_file"/>
   <addaction name="actionAdd_folder"/>
   <addaction name="separator"/>
   <addaction name="actionRename"/>
   <addaction name="actionConvert"/>
  </widget>
  <action name="actionAdd_file">
   <property name="text">
    <string>Add...</string>
   </property>
   <property name="toolTip">
    <string>Add files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionAdd_folder">
   <property name="text">
    <string>Add folder...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove_selected_files">
   <property name="text">
    <string>Remove selected files</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionViewInfo_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info panel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave_metadata">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionFilter_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter panel</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename...</string>
   </property>
   <property name="toolTip">
    <string>Rename selected files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionConvert">
   <property name="text">
    <string>Convert...</string>
   </property>
   <property name="toolTip">
    <string>Convert selected files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionViewToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionToolbarIconSmall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </action>
  <action name="actionToolbarIconsLarge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Large</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionForum">
   <property name="text">
    <string>Forum</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookInfoPanel</class>
   <extends>QWidget</extends>
   <header>.bookinfopanel.h</header>
   <container>1</container>
   <slots>
    <signal>focusLost()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BookTableView</class>
   <extends>QTableView</extends>
   <header>.booktableview.h</header>
  </customwidget>
  <customwidget>
   <class>MainToolbar</class>
   <extends>QToolBar</extends>
   <header>.maintoolbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAdd_file</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_folder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_all</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_selected_files</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRemoveSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewInfo_panel</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onViewInfoPanel(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_metadata</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSaveMetadata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFilter_panel</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onViewFilterPanel(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolFilterButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onToolFilterButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1052</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRename</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_Qt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewToolbar</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onViewToolbar(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConvert</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToolbarIconSmall</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onToolbarIconSmall()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToolbarIconsLarge</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onToolbarIconLarge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionForum</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onForumLink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddFiles()</slot>
  <slot>onAddFolder()</slot>
  <slot>onExit()</slot>
  <slot>onBookListSelectionChanged()</slot>
  <slot>onBookInfoFocusLost()</slot>
  <slot>onSelectAll()</slot>
  <slot>onRemoveSelected()</slot>
  <slot>onViewInfoPanel(bool)</slot>
  <slot>onSaveMetadata()</slot>
  <slot>onViewFilterPanel(bool)</slot>
  <slot>onToolFilterButton()</slot>
  <slot>onRename()</slot>
  <slot>onAboutQt()</slot>
  <slot>onAbout()</slot>
  <slot>onViewToolbar(bool)</slot>
  <slot>onConvert()</slot>
  <slot>onToolbarIconLarge()</slot>
  <slot>onToolbarIconSmall()</slot>
  <slot>onHelp()</slot>
  <slot>onForumLink()</slot>
 </slots>
</ui>
